iwyu.cc

```diff
@@ -4223,6 +4226,34 @@ class IwyuAstConsumer
     return HandleAliasedClassMethods(decl);
   }
 
+  // Avoid warnings for types which _should_ be already provided by the primary
+  // method declaration.
```

Comment:
Might be worth being more specific with `Avoid forward-declare warnings...`
 
Also, a micro-example here would make it easier to understand this, something like;
```
  // For example, for 'struct Derived : public Base { T1 method(const T2&); };',
  // T1 and T2 are provided by Base, and should not be forward-decl reported in
  // Derived.
```